psqlgraph/psql.py

```diff
@@ -370,17 +375,18 @@ def node_delete_system_annotation_keys(self,
                                            session=None,
                                            max_retries=DEFAULT_RETRIES,
                                            backoff=default_backoff):
-        raise NotImplementedError()
-        # with self.session_scope(session) as local:
-        #     if not node:
-        #         node = self.node_lookup_one(node_id=node_id)
-        #
-        #     if not node:
-        #         raise QueryError('Node not found')
-        #
-        #     for key in system_annotation_keys:
-        #         del node.system_annotations[key]
-        #     local.merge(node)
+        with self.session_scope(session) as local:
+            if not node:
+                node = self.node_lookup_one(node_id=node_id)
+
+            if not node:
+                raise QueryError('Node not found')
+
+            for key in system_annotation_keys:
+                del node.system_annotations[key]
```

Comment:
if I understood the comment correctly, the suggestion was to remove the commented out code. but if we actually want to implement a default behavior for removing sysans, then could we do a `pop(key, None)` here instead?